src/github.com/mozilla-services/pushgo/simplepush/emcee_store.go

```diff
@@ -544,7 +547,10 @@ func (s *EmceeStore) FetchAll(uaid string, since time.Time) ([]Update, []string,
 	keys := make([]string, 0, 20)
 
 	for _, chid := range chids {
-		key, _ := s.IDsToKey(uaid, chid)
+		key, err := s.IDsToKey(uaid, chid)
+		if err != nil {
```

Comment:
It might be clearer to revert this to `key, _ := s.IDsToKey(uaid, chid)`—or even split `IDsToKey` into two methods. In this case, we've already validated `uaid`, and likewise validate channel IDs before storing them.